src/app/pages/creator-fest/home-content/home-content.js

```diff
@@ -1,43 +1,17 @@
-import componentType from '~/helpers/controller/init-mixin';
-import css from './home-content.css';
+import React from 'react';
 import About from './about/about';
 import Why from './why/why';
+import './home-content.css';
 
-const spec = {
-    css,
-    view: {
-        classes: ['home']
-    }
-};
-
-export default class extends componentType(spec) {
-
-    onLoaded() {
-        const rData = this.data.register[0][0];
-        const registerBox = {
-            headline: rData.headline,
-            address: rData.address,
-            url: rData.button_url,
-            text: rData.button_text
-        };
-
-        this.data.page_panels.forEach((pData, i) => {
-            const model = {
-                superheadline: pData.superheading,
-                headline: pData.heading,
-                htmlBlock: pData.embed,
-                cards: pData.cards,
-                description: pData.paragraph,
-                background: pData.background_image.image
-            };
-            let SectionConstructor = Why;
-
-            if (i === 0) {
-                Object.assign(model, {registerBox});
-                SectionConstructor = About;
+export default function HomeContent({pagePanels, register}) {
+    return (
+        <div className="home">
+            <About data={pagePanels[0]} register={register} />
+            {
+                pagePanels.slice(1).map((data) =>
+                    <Why data={data} key={data.superheading} />
```

Comment:
same question as on one of the other pro, `superheating` seems like it might not be unique, maybe it'd be better to just use the index of the element here

Reply:
Sounds good to me.